packages/react-scripts/template/src/components/example/WagonWheelControl.js

```diff
@@ -22,8 +22,11 @@ export const WheelSpeedOptions = {
   },
 }
 
+const fallbackWidthValue = '740px'
+const widthDivisor = 2
```

Comment:
Sorry, but this still seems magic to me. The divisor can stay in the calculation, since it doesn't change and the variable just complicates the code. It's the 740 that seems magic. I don't know what the number means or why I would need to change it or where it comes from. Just making a var still seems magic, although now you could add a comment to explain it better I suppose.
My real question is this something that can be calculated or introspected?
If it does need to be constant like this, then we should either add a comment here, or you can put it directly back in the calculation and add a comment there (if comments work well in that context)

Reply:
And is there any value since we're already in code to do the calc here instead of in css?
```
const fallbackWidthValue = `${740 / 2}px`
```

Reply:
The most zion-like way to manage content at different sizes is use `useAtSize()`.  
https://beta.familysearch.org/frontier/zion/?path=/docs/ui-foundation-layout-responsive-content--demo

Reply:
Can't really be deduced, so I will add a comment.

Reply:
@marybeshaw Yeah, `useAtSize()` can't really be used here.